data/json/monster_factions.json

```diff
@@ -27,7 +27,7 @@
     "name": "zombie",
     "friendly": [ "slime" ],
     "neutral": [ "cult", "small_animal", "fish", "insect", "aquatic_predator" ],
-    "by_mood": [ "ant", "acid_ant", "bee", "wasp" ]
+    "by_mood": [ "ant", "acid_ant", "bee", "wasp", "spider" ]
```

Comment:
Zombies are neutral with regard to most things they can't make into zombies; the insect exceptions are social ones or otherwise obvious competitors - that have a chance against them. If you make zombies fight with other things (as will happen with by_mood), given revival zombies nearly always win. Give appropriate spiders an attack that prevents revival (via internal digestion, probably) and that might be different. (I do have to wonder if dermatiks might reasonably be another exception, since they could keep zombies as incubators.)

Reply:
Note that by "internal digestion" I'm referring to digestion happening inside the creature _being_ digested - injection of digestive enzymes, essentially. (This is sometimes referred to as "extracorporeal digestion".) It would be highly desirable for the messages from "eating a corpse" to be changeable per-species in JSON, to make this clearer for spiders.) Most suitable for this are spiders that inject a directly-damaging venom, which is already meant to represent (at least partially) digestive enzymes.